apps/ai-content-generator/package.json

```diff
@@ -23,7 +24,7 @@
     "build": "tsc && vite build --mode ${NODE_ENV:='production'}",
     "test": "vitest",
     "create-app-definition": "contentful-app-scripts create-app-definition",
-    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3iszK8Gl7aaxLvxGyCOhgA --token ${CONTENTFUL_CMA_TOKEN}",
+    "doNotDeployToProd": "contentful-app-scripts upload --ci --bundle-dir ./dist --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3iszK8Gl7aaxLvxGyCOhgA --token ${CONTENTFUL_CMA_TOKEN}",
```

Comment:
so i take it you're disabling the prod deploy here to verify that everything with the LD ff works, and then you'll re-enable it later?

Reply:
Yup! Exactly that